.travis.yml

```diff
@@ -41,6 +41,12 @@ matrix:
         - DESC="checkstyle and sevntu-checkstyle"
         - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dcobertura.skip=true"
 
+    # jacoco (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="jacoco"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -Pno-validations -Pjacoco-check -DskipTests=false"
+
     # cobertura and codecov (oraclejdk8)
```

Comment:
It seems I found how to set it up in the proper way. Instead of disabling "everything except", which is a subject to change, all the validation plugins should be disabled. Then the required properties should be reverted back to false. With this approach, it will be much easier to add additional checks later.